### PR TITLE
Added section in CONTRIBUTING.md about performance benchmarks

### PR DESCRIPTION
**Pull request by astrofrog**
I've written a mini guide over at [astropy-benchmarks](https://github.com/astropy/astropy-benchmarks#contributing-a-benchmark) and have linked to it in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ include the following:
   detailed [documentation guidelines](http://docs.astropy.org/en/latest/development/docguide.html)
   to help you out.
 
+  **Performance improvements**: if you are making changes that impact Astropy
+  performance, consider adding a performance benchmark in the
+  [astropy-benchmarks](https://github.com/astropy/astropy-benchmarks)
+  repository. You can find out more about how to do this
+  [here](https://github.com/astropy/astropy-benchmarks#contributing-a-benchmark)
```
Comment (bsipocz) [inline CONTRIBUTING.md]: maybe use something more descriptive than `here`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,16 +14,16 @@ Contributing code
 So you're interested in contributing code to Astropy? Excellent!
 
 Most contributions to Astropy are done via pull requests from GitHub users'
-forks of the [astropy repository](https://github.com/astropy/astropy). If 
+forks of the [astropy repository](https://github.com/astropy/astropy). If
 you're new to this style of development, you'll want to read over our [development workflow](http://docs.astropy.org/en/latest/development/workflow/development_workflow.html).
```
Comment (pllim) [inline CONTRIBUTING.md]: Tsk tsk. While we are editing this page, perhaps break this line into shorter lines? :sweat_smile:

File: CONTRIBUTING.md (hunk 1 above)
Comment (astrofrog) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (astrofrog) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**